src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagMercury.java

```diff
@@ -0,0 +1,42 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEng
+ */
+@Test(groups = "AdsFliteTagMercury")
+public class TestAdsFliteTagMercury extends MobileTestTemplate {
+
+  private static final String FLITE_CSS_SELECTOR_MERCURY = ".widget-flite";
+  private static final String
+      FLITE_ERROR_CSS_SELECTOR_MERCURY =
+      ".ember-view.article-content.mw-content .error";
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteTagMercury"
+  )
+  @Execute(onWikia = "adtest")
```

Comment:
Don't like the annotations because you don't know from first glance where the feature was initialised :-1: